<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SAPTEST_A00" name="Navegacion_Header2" tests="4" failures="1" timestamp="17 Aug 2017 14:16:03 GMT" time="8.354" errors="0">
  <testcase name="NuevosAlumnos" time="1.154" classname="Salle.TestClasses.SalleHeader"/>
  <testcase name="Alumnos" time="1.335" classname="Salle.TestClasses.SalleHeader"/>
  <testcase name="Alumni" time="1.23" classname="Salle.TestClasses.SalleHeader"/>
  <testcase name="Empresas" time="0.678" classname="Salle.TestClasses.SalleHeader">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
	at Salle.TestClasses.SalleHeader.Empresas(SalleHeader.java:96)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Empresas -->
</testsuite> <!-- Navegacion_Header2 -->
